network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -295,6 +310,45 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         blank=True,
     )
 
+    # Un-editable voting fields. Don't add these to the content_panels.
+    current_total = models.IntegerField(default=0)
+    current_votecount = models.IntegerField(default=0)
+    vote_bin = models.OneToOneField(
```

Comment:
This is not a single bin, let's go with "votes":
```suggestion
    votes = models.OneToOneField(
```